client/service/src/builder.rs

```diff
@@ -1139,7 +1137,7 @@ ServiceBuilder<
 			let system_info = sc_rpc::system::SystemInfo {
 				chain_name: chain_spec.name().into(),
 				impl_name: config.impl_name.into(),
-				impl_version: config.impl_version.into(),
+				impl_version: config.impl_version.into(), // TODO: this is now using full version instead of just version. Is it correct?
```

Comment:
Just to be sure everybody is aware: the impl_version use to be the "short" version (ex: 1.0.0) but now it is the full version (ex: 2.0.0-alpha.2-b950f731c-x86_64-linux-gnu)

Reply:
cc @tomusdrw

Reply:
I think that's fine, just make sure to mention it to @jacogr and include in release notes.

Reply:
he said all fine ^_^